client/app/components/Parameters.less

```diff
@@ -1,32 +1,24 @@
 @import '../assets/less/ant';
 
-.drag-handle {
-  background: linear-gradient(90deg, transparent 0px, white 1px, white 2px)
-              center,
-              linear-gradient(transparent 0px, white 1px, white 2px) center, #111111;
-  background-size: 2px 2px;
-  display: inline-block;
-  width: 6px;
-  height: 36px;
-  vertical-align: bottom;
-  margin-right: 5px;
-  cursor: move;
-}
-
 .parameter-block {
   display: inline-block;
   background: white;
   padding: 0 12px 6px 0;
   vertical-align: top;
```

Comment:
Add `z-index: 1` so this don't happen:

<img width="271" alt="Screen Shot 2019-09-30 at 17 33 25" src="https://user-images.githubusercontent.com/486954/65888591-75b02600-e3a8-11e9-93de-dc0cfc8aec71.png">

Reply:
Done :+1: